src/status_im/ethereum/json_rpc.cljs

```diff
@@ -35,7 +35,10 @@
    "status_joinPublicChat" {}
    "status_chats" {}
    "status_startOneOnOneChat" {}
-   "status_removeChat" {}})
+   "status_removeChat" {}
```

Comment:
Maybe worth a clearer naming convention?

Reply:
yes but that's on status-go side though